includes/data-stores/class-wc-admin-notes-data-store.php

```diff
@@ -203,6 +200,22 @@ private function save_actions( &$note ) {
 	 */
 	public function get_notes() {
 		global $wpdb;
-		return $wpdb->get_results( "SELECT note_id, title, content FROM {$wpdb->prefix}woocommerce_admin_notes order by note_id ASC;" );
+		return $wpdb->get_results( "SELECT note_id, title, content FROM {$wpdb->prefix}woocommerce_admin_notes ORDER BY note_id ASC;" );
+	}
+
+	/**
+	 * Find all the notes with a given name.
+	 *
+	 * @param string $name Name to search for.
+	 * @return array An array of matching note ids.
+	 */
+	public function get_notes_with_name( $name ) {
+		global $wpdb;
+		return $wpdb->get_col(
+			$wpdb->prepare(
+				"SELECT note_id FROM {$wpdb->prefix}woocommerce_admin_notes WHERE name = %s ORDER BY note_id ASC;",
```

Comment:
Same question as `get_notes()` - should we order by date created by default or do you think ID is better?

Reply:
Yeah - I think id will be more performant than string comparison. Great question though - I did spend a little time thinking about using date created when i wrote that